examples/experimental/onnx/semantic_segmentation/README.md

```diff
@@ -0,0 +1,62 @@
+# Semantic Segmentation sample
+
+This sample shows an example of quantization of segmentation models. The used datasets are CamVid, Mapillary.
+
+## Install
+
+Please, install the requirements for ONNX Post-Training Quantization of NNCF.
+
+Install requirements
+
+```
+pip install -r <nncf dir>/nncf/experimental/onnx/requirements.txt
+pip install -r <nncf dir>/examples/experimental/onnx/requirements.txt
+```
+
+## Getting the quantized model
```

Comment:
"Getting the model" step is required to guide where to take it. This step can be called, like "Quantizing the model"